docs/src/main/sphinx/connector/system.rst

```diff
@@ -50,6 +50,46 @@ that can be set when creating a new schema.
 The table properties table contains the list of available properties
 that can be set when creating a new table.
 
+.. _system_metadata_materialized_views:
+
+``metadata.materialized_views``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The materialized views table contains the following information about all
+:ref:`materialized views <sql-materialized-views-management>`:
+
+.. list-table:: Metadata for materialized views
+  :widths: 50, 50
```

Comment:
We may want to change widths to `30, 70` in follow-up PR.
<img width="847" alt="Screen Shot 2021-10-02 at 11 16 23" src="https://user-images.githubusercontent.com/6237050/135700829-c84c845b-054b-45c4-ae24-fd03e4ae5ae5.png">

<img width="852" alt="Screen Shot 2021-10-02 at 11 17 55" src="https://user-images.githubusercontent.com/6237050/135700833-0af12c32-43f6-4d90-b564-8a3a8dc6d2fb.png">

Reply:
Sure ... did you want to send that PR and get it merged before 363 @ebyhr ?